mimesniff/mime-type-sniffing/scriptable-mime-types.window.js

```diff
@@ -0,0 +1,57 @@
+// META: timeout=long
+// META: script=/common/subset-tests.js
+// META: variant=?1-738
+// META: variant=?739-1476
+// META: variant=?1477-2214
+// META: variant=?2215-2952
+// META: variant=?2953-3690
+// META: variant=?3691-last
+
+setup({
+  "explicit_timeout": true,
```

Comment:
Is this needed, and does it really work together with the "META: timeout=long" setting? Per https://web-platform-tests.org/writing-tests/testharness-api.html#setup-properties there should be some call to `timeout()` when using this setup.

Reply:
I tried many different combinations of timeouts to get things to not time out early. Chrome does not fire a load event when an iframe downloads, so I have to explicitly time that out. And having multiple iframes load at once causes a pause that the runner doesn't like and/or that causes later tests to time out.

If you can find a different combination of settings that doesn't time out before all the tests are run, I'm all ears. But I spent way more time than I wanted to trying to figure this one out.

Reply:
I don't have good suggestions here, but it does seem like the Chrome model is the way to go.

The one thing that might help here is an API for testing, but that would be its own somewhat significant undertaking. cc @jgraham

Reply:
Unless this is manifesting as an obvious detriment now, I feel like this is something that can be resolved at a later date?